styles/spotify-web/catppuccin.user.css

```diff
@@ -166,6 +166,12 @@
     .pHrwZOFBdT8FNXnmcPPI {
       background: @crust !important;
     }
+
+    /* some borders */
```

Comment:
This doesn't clarify anything - we already know this is theming a border. The question is where. Add details!

Reply:
does the new comment work?